capture-sdk/sdk/src/main/java/net/gini/android/capture/network/Result.java

```diff
@@ -12,14 +12,16 @@
 public class Result {
 
     private final String giniApiDocumentId;
+    private final String giniApiDocumentFilename;
 
```

Comment:
[Suggestion] Are these breaking changes? As I see, these are public classes which might be used by clients. Please let me know what you think.

Reply:
I'm not sure. We can create extra constructor with some empty text by default. 
Customers which will not use transaction docs will not be affected then